contracts/JBPaymentTerminalStore.sol

```diff
@@ -185,34 +175,49 @@ contract JBPaymentTerminalStore is IJBPaymentTerminalStore, ReentrancyGuard {
 
   /**
     @notice
-    The amount of overflowed tokens that can be reclaimed by the specified number of tokens.
+    The current amount of overflowed tokens from a terminal that can be reclaimed by the specified number of tokens.
 
     @dev 
     If the project has an active funding cycle reconfiguration ballot, the project's ballot redemption rate is used.
 
     @dev
-    The reclaimable overflow is represented as a fixed point number with 18 decimals.
+    The current reclaimable overflow is returned in terms of the specified terminal's currency.
+
+    @dev
+    The reclaimable overflow is represented as a fixed point number with the same amount of decimals as the specified terminal.
 
-    @param _terminal The terminal from which the overflow is being calculated.
-    @param _projectId The ID of the project to get a reclaimable amount for.
+    @param _terminal The terminal from which the reclaimable amount would come.
+    @param _projectId The ID of the project to get the reclaimable overflow amount for.
     @param _tokenCount The number of tokens to make the calculation with, as a fixed point number with 18 decimals.
+    @param _useTotalOverflow A flag indicating whether the overflow used in the calculation should be summed from all of the project's terminals. If false, overflow should be limited to the amount in the specified `_terminal`.
 
     @return The amount of overflowed tokens that can be reclaimed.
   */
-  function reclaimableOverflowOf(
+  function currentReclaimableOverflowOf(
     IJBPaymentTerminal _terminal,
     uint256 _projectId,
-    uint256 _tokenCount
+    uint256 _tokenCount,
+    bool _useTotalOverflow
```

Comment:
why are we not reading it from the funding cycle metadata?

Reply:
In this case cus the outside world can view what the reclaimable overflow would be in either case. Otherwise the only way to "know" what the value would be is to reconfigure, or do the calculation manually.